code/game/turfs/space/space.dm

```diff
@@ -154,17 +154,19 @@
 			DT = locate(tx, ty, destination_z)
 
 		var/atom/movable/AM = A.pulling
-		A.forceMove(DT)
 		if(AM)
-			var/turf/T = get_step(A.loc,turn(A.dir, 180))
-			AM.forceMove(T)
+			A.forceMove(get_step(DT,A.dir))
```

Comment:
Now you're teleporting people to different places depending on if they're pulling something. If the issue atm is that the thing being pulled is being teleported by the world edge just make the world edge not teleport things being pulled. Treat the root problem.

Ask for help if you don't get how a bug should get fixed; we're available on #coderbus on Rizon